src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -173,8 +180,15 @@ private void setListMethodFields(
     PageStreamingConfig pageStreaming = context.getMethodConfig().getPageStreaming();
     TypeRef resourceType = pageStreaming.getResourcesField().getType();
     String resourceTypeName = context.getTypeTable().getAndSaveNicknameForElementType(resourceType);
+    String resourceFieldName =
+        context.getNamer().getFieldGetFunctionName(pageStreaming.getResourcesField());
```

Comment:
This is an interesting case. Go doesn't fall into the abstraction very well, since it uses upper camel for exported (public) fields and functions and lower camel for private ones.

I used the function name method to mean "how other people get to the field" which, in Go, means upper camel. I'm creating dedicated `getFieldName` for now with upper camel, since that seems to be what we usually need.